{"name": "Login with wrong email", "status": "passed", "description": "Signing in with the following credentials email : kamelmohsenkamel & password: Kimo2010", "attachments": [{"name": "SUCCESS: Login failed with incorrect credentials", "source": "65e4e958-18bd-42d7-9281-f4b3cdea7da0-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "69cb7a8a-64ca-415b-942e-22e987023eee-attachment.txt", "type": "text/plain"}], "start": 1585784719135, "stop": 1585784726995, "uuid": "ccd9788e-eb04-4b87-818e-c0e6e4be7e86", "historyId": "e7b002042516d1a2b8ca85621381b84e", "testCaseId": "dc0657a7bdd61946fb3cc87809c963a8", "fullName": "Web_Testing.Tests.signin_tests.TestSignin#test_case_6", "labels": [{"name": "parentSuite", "value": "End to End testing"}, {"name": "severity", "value": "blocker"}, {"name": "story", "value": "Failing login"}, {"name": "suite", "value": "Login Tests"}, {"name": "subSuite", "value": "Login with wrong email"}, {"name": "feature", "value": "Log in Tests"}, {"name": "tag", "value": "Login"}, {"name": "tag", "value": "Do"}, {"name": "host", "value": "LAPTOP-8H6HECEC"}, {"name": "thread", "value": "35856-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Web_Testing.Tests.signin_tests"}]}